packages/build/tests/error/snapshots/tests.mjs.md

```diff
@@ -3728,6 +3728,10 @@ Generated by [AVA](https://avajs.dev).
     ␊
       Error message␊
       Error: test␊
+    ␊
+      Plugin details␊
+      Package:        ./plugin.js␊
+      Repository:     test␊
```

Comment:
The snapshots of this test and a few others show additional `Plugin details` due to the addition of a `package.json` in the test fixtures.